### PR TITLE
Improved docker.sh script

### PR DESCRIPTION
**Pull request by partikus**
Got rid of advanced case statement.
Removed useless sleeps.
Updated README.md

Should be merge after #24

### CODE REVIEW

**Comment by partikus** [comment]
@jhryniuk @dominikpgs @tarnawski @kubawerlos CR please, 

should be merged after #24

File: docker.sh [hunk 1]
```diff
@@ -1,19 +1,24 @@
 #!/usr/bin/env bash
 
-TASK_NAME=$1;
+PROJECT_TASK_NAME=$1;
+PROJECT_PHP_VERSION=${2:-71};
```
Comment (jhryniuk) [inline docker.sh]: why `TASK_NAME` has been divided to `PROJECT_TASK_NAME` and `PROJECT_PHP_VERSION`?

Comment (partikus) [reply docker.sh]: @jhryniuk please take a look [here](https://github.com/partikus/php-docker-skeleton/blob/681b9e5f9f60a830a1d6e252bd859b630d61628c/README.md) how new syntax look like

**Review by jhryniuk — APPROVED** [review]

**Comment by partikus** [comment]
closed in favor of #24

**Pull request closed without merging**